test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14107,6 +14107,31 @@ def enable_cgroup_cset(self):
                               self.hostname)
             raise AssertionError('cgroup subsystems not mounted')
 
+    def cleanup_homedir(self, user):
+        self.logger.info('Cleaning %s\'s home directory' % (str(user)))
+        runas = PbsUser.get_user(user)
+        hostname = self.hostname
+        ret = self.du.run_cmd(hostname, cmd=['echo', '$HOME'], sudo=True,
+                              runas=runas, logerr=False, as_script=True)
+        if ret['rc'] == 0:
+            path = ret['out'][0].strip()
+        else:
+            return None
+
+        ftd = []
+        files = self.du.listdir(hostname, path=path, runas=user)
+        bn = os.path.basename
+        ftd.extend([f for f in files if bn(f).startswith('PtlPbs')])
+        ftd.extend([f for f in files if bn(f).startswith('STDIN')])
+        ftd.extend([f for f in files if re.match('.*\.o.*', f)])
+        ftd.extend([f for f in files if re.match('.*\.e.*', f)])
+
+        if len(ftd) > 1000:
```

Comment:
This removal of files is done in test_runner() in PbsTestSuite.  You should probably remove the duplicate code there if you are removing it here.